pkg/cmd/server/origin/master.go

```diff
@@ -828,24 +826,63 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 		"clusterRoleBindings":   clusterRoleBindingStorage,
 		"clusterRoles":          clusterRoleStorage,
 
-		"clusterResourceQuotas":        clusterResourceQuotaStorage,
-		"clusterResourceQuotas/status": clusterResourceQuotaStatusStorage,
-		"appliedClusterResourceQuotas": appliedclusterresourcequotaregistry.NewREST(
-			c.ClusterQuotaMappingController.GetClusterQuotaMapper(), c.Informers.ClusterResourceQuotas().Lister(), c.Informers.KubernetesInformers().Namespaces().Lister()),
-
 		"roleBindingRestrictions": roleBindingRestrictionStorage,
 	}
 
+	storage[securityapiv1.SchemeGroupVersion] = map[string]rest.Storage{
+		"podSecurityPolicyReviews":            podSecurityPolicyReviewStorage,
+		"podSecurityPolicySubjectReviews":     podSecurityPolicySubjectStorage,
+		"podSecurityPolicySelfSubjectReviews": podSecurityPolicySelfSubjectReviewStorage,
+	}
+
+	storage[projectapiv1.SchemeGroupVersion] = map[string]rest.Storage{
+		"projects":        projectStorage,
+		"projectRequests": projectRequestStorage,
+	}
+
+	storage[deployapiv1.SchemeGroupVersion] = map[string]rest.Storage{
+		"deploymentConfigs":             deployConfigStorage,
+		"deploymentConfigs/scale":       deployConfigScaleStorage,
+		"deploymentConfigs/status":      deployConfigStatusStorage,
+		"deploymentConfigs/rollback":    deployConfigRollbackStorage,
+		"deploymentConfigs/log":         deploylogregistry.NewREST(configClient, kclient, c.DeploymentLogClient(), nodeConnectionInfoGetter),
+		"deploymentConfigs/instantiate": dcInstantiateStorage,
+	}
+
+	storage[templateapiv1.SchemeGroupVersion] = map[string]rest.Storage{
+		"processedTemplates": templateregistry.NewREST(),
+		"templates":          templateStorage,
+	}
+
+	storage[imageapiv1.SchemeGroupVersion] = map[string]rest.Storage{
+		"images":               imageStorage,
+		"imagesignatures":      imageSignatureStorage,
+		"imageStreams/secrets": imageStreamSecretsStorage,
+		"imageStreams":         imageStreamStorage,
+		"imageStreams/status":  imageStreamStatusStorage,
+		"imageStreamImports":   imageStreamImportStorage,
+		"imageStreamImages":    imageStreamImageStorage,
+		"imageStreamMappings":  imageStreamMappingStorage,
+		"imageStreamTags":      imageStreamTagStorage,
+	}
```

Comment:
Have a look at [pkg/registry/batch/rest/storage_batch.go](https://github.com/kubernetes/kubernetes/blob/5e29e1ee05556800cd2fe0dc5341d5de54b4ab47/pkg/registry/batch/rest/storage_batch.go), for example, in k8s, it would be nice to follow this path. Although it'll be a bit harder, since we have direct dependencies between certain controllers and direct access to storage.